oem/ibm/pfw-sms-utils/meson.build

```diff
@@ -0,0 +1,14 @@
+libpam = cpp.find_library('pam', required: true)
+try_pfw_sms_deps = [libpam]
+
+executable(
+  'try-pfw-sms-menu',
+  'try_pfw_sms_ibm.cpp',
```

Comment:
Please remove these try files and dependencies as these can be enabled with Gtest in another commit

Reply:
Yes, I will remove this.